### DISCUSSION TITLE
Working with Pasted Content in JavaScript

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#27] [Comments]
# Working with Pasted Content in JavaScript

How to handle pasted content with vanilla JavaScript.

https://www.raymondcamden.com/2024/07/03/working-with-pasted-content-in-javascript

**Comment by fitsum** [comment]
enjoyed the post
also liking this giscus widget 👌🏾

**Reply by cfjedimaster** [reply]
It was fairly painless to add. I don't get a lot of comments, but I'm still happy I added it.

**Comment by SimonKuznetsov** [comment]
Hi! Thanks for very useful article
